rpcserver.go

```diff
@@ -510,17 +510,47 @@ func (s *rpcServer) InitAccount(ctx context.Context,
 			"must be specified")
 	}
 
-	// Determine the desired transaction fee.
-	confTarget := req.GetConfTarget()
-	if confTarget < 1 {
-		return nil, fmt.Errorf("confirmation target must be " +
-			"greater than 0")
+	var feeRate chainfee.SatPerKWeight
+
+	switch {
+	case req.GetFeeRateSatPerKw() != 0:
+		feeRate = chainfee.SatPerKWeight(req.GetFeeRateSatPerKw())
+	case req.GetConfTarget() > 0:
+		// Determine the desired transaction fee.
+		confTarget := req.GetConfTarget()
+
+		resp, err := s.QuoteAccount(context.Background(),
```

Comment:
Since we're already checking that the conf target is larger than 0, we can just call the `s.accountManager.QuoteAccount()` here directly instead of doing a call on the RPC server level.